modules/install-openshift-common-terms.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/index.adoc
+
+:_content-type: REFERENCE
+[id="install-openshift-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the installation content. These terms help you understand installation effectively.
+
+Bootstrap node::
+A temporary machine that runs a minimal Kubernetes configuration to deploy the {product-title} control plane. It is deleted at the end of the installation.
+
+Control plane::
+ It is a container orchestration layer that exposes the API and interfaces to define,deploy, and manage the lifecycle of containers. Also known as control plane machines.
+
+Compute node::
+Nodes that are responsible for executing workloads for cluster users. Also known as worker nodes.
+
+Disconnected installation::
+There are situations where parts of a data center might not have access to the internet, even through proxy servers. You can still install the{product-title} in these environments, but you must download the required software and images and make them available to the disconnected environment.
+
+The {product-title} installation program::
+A program that provisions the infrastructure and deploys a cluster.
+
+Installer-provisioned infrastructure::
+The installation program deploys and configures the infrastructure that the cluster runs on.
+
+Ignition config files::
+A file that Ignition uses to configure Red Hat Enterprise Linux CoreOS (RHCOS) during operating system initialization. The installation program generates different Ignition config files to initialize bootstrap, control plane, and worker nodes.
```

Comment:
> The installation program generates different Ignition config files to initialize bootstrap, control plane, and worker nodes.

I'm not sure if this description is accurate, b/c:
* The control plane, and worker nodes' ignitions are not _files_ but _endpoints_ (a URL). 
* Are control plane, and worker nodes' ignitions are generated by installer ? please @patrickdillon help to confirm,  thanks.

Reply:
The installer does create ignition config files for bootstrap, control plane and compute nodes. You are correct that the files for control plane and compute are stub ignition configs that only contain endpoints. I think the current wording of the pr is fine, especially for an introductory audience this is intended for.